packages/common-integration-tests/src/storage/storage.ts

```diff
@@ -253,21 +109,125 @@ export function runStorageServiceTests(
         }
       }
     });
-    it('should successfully make two of the same requests after 5s retry timeout', async () => {
-      await WithStore(storageClient, testingStoreName, async () => {
-        const key = v4();
-        const getResponse1 = await storageClient.get(testingStoreName, key);
-        expectWithMessage(() => {
-          expect(getResponse1.type).toEqual(StorageGetResponse.NotFound);
-        }, `expected NotFound, received ${getResponse1.toString()}`);
+  });
 
-        await sleep(5000);
+  // Data plane tests should use the shared integration test store
+  // with the BeforeAll setup and AfterAll teardown
+  describe('#store get put and delete', () => {
```

Comment:
all the storage data plane tests should be in this describe block and should use the shared store with unique keys